CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # CHANGELOG
 ## X.X.X - 2022-XX-XX
 
+## 19.1.1 - 2022-01-31
+This release has several bug fixes, disables card saving by default in payment sheet, and relays the original error message on failed payments in payment sheet. 
```

Comment:
nit: This says several bug fixes but there is only one bugfix listed.  Consider removing this summary entirely - I like the idea but this release only has 3 changes

Reply:
yeah I wasn't sure if we needed a summary if the changes were so small.